scheduler/src/cook/compute_cluster.clj

```diff
@@ -62,6 +62,18 @@
      :mesos-framework-id framework-id
      :db-id (or cluster-entity-id (get-mesos-cluster-entity-id (d/db conn) mesos-cluster))}))
 
+(def default-cluster-name-for-legacy (atom nil))
+
+(defn get-default-cluster-name-for-legacy
+  "What cluster name to put on for legacy jobs when generating their compute-cluster."
+  []
+  @default-cluster-name-for-legacy)
```

Comment:
Can we get rid of the atom and just have this read the config? This shouldn't change over the lifetime of the process.

Reply:
Done. I'll make a parallel change in the new patch. :)